### PR TITLE
Feature - Display Relationships in Info Command Output

### PR DESCRIPTION
**Pull request by dmccoystephenson**
The changes in this PR allow players to view the vassals, allies and enemies of a faction through the 'mf info' command.

### CODE REVIEW

**Comment by dmccoystephenson** [comment]
closes #1600

**Comment by alyphen** [comment]
Code looks good, just needs translations.
British is the same as American for these.

We can ask the Discord for German/French or look at existing translations to find which words have been used there. This shouldn't be a complex one since it's just singular words.

**Comment by dmccoystephenson** [comment]
French translations are required for the following:

CommandFactionInfoVassalsTitle=Vassals:
CommandFactionInfoAlliesTitle=Allies:
CommandFactionInfoEnemiesTitle=Enemies:

File: src/main/resources/lang/lang_fr_FR.properties [hunk 1]
```diff
@@ -1018,3 +1018,6 @@ FactionDisbandedZeroPowerNotificationBody=Votre faction, {0}, a 
 Teleporting=T�l�portation dans {0} secondes...
 TeleportationCancelled=T�l�portation annul�e.
 PowerIncreased=Vous vous sentez plus fort. Votre pouvoir a augment� de {0}.
+CommandFactionInfoVassalsTitle=Vassaux:
+CommandFactionInfoAlliesTitle=Alli�s:
```
Comment (alyphen) [inline src/main/resources/lang/lang_fr_FR.properties:1022]: Accent hasn't been committed properly

File: src/main/kotlin/com/dansplugins/factionsystem/command/faction/info/MfFactionInfoCommand.kt [hunk 2]
```diff
@@ -146,6 +146,28 @@ class MfFactionInfoCommand(private val plugin: MedievalFactions) : CommandExecut
                         }
                     )
                 }
+
+                // send vassals information
+                val vassals = plugin.services.factionRelationshipService.getVassals(faction.id).mapNotNull(factionService::getFaction)
+                if (vassals.isNotEmpty()) {
+                    sender.sendMessage("${BukkitChatColor.WHITE}${plugin.language["CommandFactionInfoVassalsTitle"]}") // TODO: add translation
```
Comment (alyphen) [inline src/main/kotlin/com/dansplugins/factionsystem/command/faction/info/MfFactionInfoCommand.kt:153]: Todos should be removed

**Review by alyphen — CHANGES_REQUESTED** [review]

File: src/main/resources/lang/lang_fr_FR.properties (hunk 1 above)
Comment (dmccoystephenson) [reply src/main/resources/lang/lang_fr_FR.properties:1022]: I'm not sure how to resolve this.

File: src/main/kotlin/com/dansplugins/factionsystem/command/faction/info/MfFactionInfoCommand.kt (hunk 2 above)
Comment (dmccoystephenson) [reply src/main/kotlin/com/dansplugins/factionsystem/command/faction/info/MfFactionInfoCommand.kt]: These have been removed.

**Pull request closed without merging**